### PR TITLE
feat(TagsInput): allow entering custom tags when in suggestions mode

### PR DESCRIPTION
**Pull request by plagoa**

### CODE REVIEW

File: packages/core/src/components/TagsInput/TagsInput.tsx [hunk 1]
```diff
@@ -476,15 +482,35 @@ export const HvTagsInput = forwardRef<HTMLUListElement, HvTagsInputProps>(
             case "ArrowDown":
               getSuggestions(0)?.focus();
               break;
+            case "Enter":
+              if (
+                canShowSuggestions &&
+                suggestionsLoose &&
+                suggestionValidation?.(tagInput)
```
[the remark this answers is not in the document]
Comment (plagoa) [reply packages/core/src/components/TagsInput/TagsInput.tsx:489]: Yep, well spotted. I'll update this.

**Comment by plagoa** [comment]
@MEsteves22 updated the PR

File: packages/core/src/components/TagsInput/TagsInput.stories.tsx [hunk 2]
```diff
@@ -452,3 +452,84 @@ export const Suggestions: StoryObj<HvTagsInputProps> = {
...
+      }
+      setOptions((prev) => [...prev, val]);
+      setStatus("valid");
+      setStatusMsg("");
+      return true;
+    };
+
+    return (
+      <>
+        <HvTypography />
```
Comment (MEsteves22) [inline packages/core/src/components/TagsInput/TagsInput.stories.tsx:515]: We can remove these components since they are not needed.

**Review by MEsteves22 — APPROVED** [review]

File: packages/core/src/components/TagsInput/TagsInput.stories.tsx (hunk 2 above)
Comment (plagoa) [reply packages/core/src/components/TagsInput/TagsInput.stories.tsx]: Done

**Review by francisco-guilherme — APPROVED** [review]

**Pull request merged**